datalab/mlalpha/_util.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import os
+import shutil
+import subprocess
+import tempfile
+
+
+def package_and_copy(package_root_dir, setup_py, out_dir):
+  """Repackage an CloudML package and copy it to a staging dir.
```

Comment:
maybe allow caller to set output name? If I run 10 training jobs, why should we make 10 copies?